tensorflow/lite/delegates/xnnpack/reshape_tester.cc

```diff
@@ -34,15 +34,40 @@ limitations under the License.
 namespace tflite {
 namespace xnnpack {
 
-void ReshapeTester::Test(TfLiteDelegate* delegate) const {
+template <class T>
```

Comment:
Add specialization for Test<float>, as it can't use `uniform_int_distribution`

Reply:
For some reason the float unittests still passed even with this code, but I updated the PR now to add a float specialization.